app/models/comment.rb

```diff
@@ -1,9 +1,17 @@
 class Comment < ApplicationRecord
 
-   include ActiveModel::Validations
+ validates :body, presence: true, length: { in: 5..500 }
 
-   validates :body, presence: true, length: { in: 5..500 }
+ belongs_to :article, counter_cache: true
+ belongs_to :user
+ has_many :opinions
+
+
+ def rating_sum
+   array = opinions.map do |o|
+     o.opinion
+   end
+   array.sum
+ end
```

Comment:
To można zrobić wbudowaną metodą.

Reply:
A tak, mam ją, zapomniałam tej skasować :)